frontend/src/scenes/products/Products.tsx

```diff
@@ -105,47 +56,37 @@ export function ProductCard({
 
     return (
         <LemonCard
-            className={clsx('flex gap-4', vertical ? 'flex-col max-w-80' : 'items-center', className)}
+            data-attr={`${product.type}-onboarding-card`}
+            className={clsx('flex justify-center cursor-pointer', vertical ? 'flex-col' : 'items-center', className)}
             key={product.type}
+            onClick={() => {
+                setIncludeIntro(false)
+                if (!onboardingCompleted) {
+                    const includeFirstOnboardingProductOnUserProperties = user?.date_joined
+                        ? new Date(user?.date_joined) > new Date('2024-01-10T00:00:00Z')
+                        : false
+                    reportOnboardingProductSelected(product.type, includeFirstOnboardingProductOnUserProperties)
+                    getStartedActionOverride && getStartedActionOverride()
+                }
+                router.actions.push(urls.onboarding(product.type))
+            }}
         >
-            <div className="flex">
-                <div>
-                    <div className="bg-mid rounded p-2">{getProductIcon(product.icon_key, 'text-2xl')}</div>
+            {onboardingCompleted && (
+                <div
+                    className="relative"
+                    onClick={(e) => {
+                        e.stopPropagation()
+                        router.actions.push(getProductUri(product.type as ProductKey, featureFlags))
+                    }}
+                >
+                    <Tooltip title="You've already set up this product. Click to return to this product's page.">
```

Comment:
But what if they want to view setup instructions again?

Reply:
This click target only affects the green check icon. Clicking anywhere else in the body of the card will take them through the onboarding flow again.

Reply:
Ohh I see, okie doke